integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/property/PropertyIntegrationDefaultScopeRemovePropertiesTestCase.java

```diff
@@ -0,0 +1,148 @@
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123.123"),
+                "Float Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Long (default scope)")
+    public void testLongVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyLongDefaultRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123123123"),
+                "Long Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Short (default scope)")
+    public void testShortVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyShortDefaultRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 12"),
+                "Short Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type OM (default scope)")
+    public void testOMVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyOMDefaultRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = OMMMMM"),
+                "OM Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    /**
+     * The method that checks whether the particular
+     * match string is available in the sysytem logs
+     */
+    private boolean isMatchFound(String matchStr) throws Exception {
+        boolean isSet = false;
+        LogEvent[] logs = logViewer.getAllSystemLogs();
+        int size = logs.length;
+        for (int i = size-1; i >= 0; i--) {
+            if (logs[i].getMessage().contains(matchStr)) {
+                for (int j = i; j >= 0; j--) {
+                    if (logs[j].getMessage().contains("symbol = null")) {
+                        isSet = true;
+                        break;
+                    }
+                }
+                break;
+            }
+        }
+        return isSet;
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy() throws Exception {
+        cleanup();
+    }
+}
```

Comment:
Please check all the files and make sure EoL is kept

Reply:
Fixed with #2148